invenio_app_ils/permissions.py

```diff
@@ -141,11 +141,9 @@ def __init__(self, record):
 
 def views_permissions_factory(action):
     """Return ILS views permissions factory."""
-    is_authenticated_user = [
-        "circulation-loan-request",
-        "patron-loans",
-        "bulk-loan-extension",
-    ]
+    is_authenticated_user = current_app.config.get(
+        "ILS_AUTHENTICATED_USER_PERMISSIONS", []
```

Comment:
well done for caring for flexibility of the configuration!
In this case it is a bit overkill though, because we already had the config:
```
ILS_VIEWS_PERMISSIONS_FACTORY = views_permissions_factory
```